bcgov/mds
# MDS-3982 - Manually Edit NoW Dates

# Main
- Ability to edit the NoW progress dates for specific users
- ability to edit NoW delay dates for specific users
- ability to edit the verification date
- ability to edit the decision date

- Dates cannot be set in the future, and there is logic to ensure the following: 
![Screen Shot 2021-11-22 at 11 24 42 AM](https://user-images.githubusercontent.com/25125247/142922702-b693cecd-e55f-4645-bf6c-b858e4752f04.png)


# Other
- hours/minutes/seconds has been removed from the "ShowDurationText" function
- Added a new date field to capture editable import and decision dates
- The original dates cannot be edited.

# How to test

- Navigate to a NoW record - Application tab, and edit dates in the "Application Progress Tracking" section 
- If you do not have permissions, (ie cannot see the pencit button) let me know

# Note
![Screen Shot 2021-11-22 at 11 26 25 AM](https://user-images.githubusercontent.com/25125247/142922899-5df1dd20-ba31-4c2f-bc1e-8463353cab1e.png)
s
